polaris-react/src/components/CalloutCard/CalloutCard.tsx

```diff
@@ -86,7 +86,9 @@ export function CalloutCard({
                   {title}
```

Comment:
CalloutCard title font-size is inconsistent with Card/LegacyCard.
The other two use `--p-font-size-325` but CalloutCard has `--p-font-size-350`.
It doesn't come up in the diff for me to add a suggestion but I recommend updating the variant on the Text component wrapping title to `headingSm` instead of `headingMd`.

Reply:
Agreed and done ✅